geocity/apps/accounts/users.py

```diff
@@ -41,6 +41,11 @@ def get_administrative_entities_associated_to_user_as_list(user):
     )
 
 
+# A trusted user is an authenticated user that is associated at least with one administrative entity
+def is_user_trusted(user):
+    return len(get_administrative_entities_associated_to_user(user)) > 0
```

Comment:
Rebase on main and use :` return len(AdministrativeEntity.associated_to_user(user)) > 0`